# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/frontend/StencilXdr.cpp
# Commit: f505b8fb8f8b
# Full Hash: f505b8fb8f8b7a8784be192ce6a36dc2278095cb
# Author: Tooru Fujisawa <arai_a@mac.com>
# Date: 2020-11-14 09:46:25
# Regressor Bug: 1675241
# File Overlap Count: 1
# Description:
#   Bug 1675241 - Part 8: Use TaggedParserAtomIndex in ScriptStencil.functionAtom. r=tcampbell
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D96915
# ==============================================================================

diff -r 3749d6d2325d -r f505b8fb8f8b js/src/frontend/StencilXdr.cpp
--- a/js/src/frontend/StencilXdr.cpp	Sat Nov 14 00:03:00 2020 +0000
+++ b/js/src/frontend/StencilXdr.cpp	Sat Nov 14 00:03:07 2020 +0000
@@ -275,7 +275,7 @@
   }
 
   if (xdrFlags & (1 << uint8_t(XdrFlags::HasFunctionAtom))) {
-    MOZ_TRY(XDRParserAtom(xdr, &stencil.functionAtom));
+    MOZ_TRY(XDRTaggedParserAtomIndex(xdr, &stencil.functionAtom));
   }
 
   return Ok();